index.html

```diff
@@ -66,15 +66,17 @@ <h3 class="form-label">Minutes</h3>
                   <input type="text" class="minutes-input" id="minutesInput"/>
                   <div class="category-warning warning-minutes hidden">
                     <img class="error-icon" src="./assets/warning.svg" alt="Error">
-                    <p>A number is required.</p>
+                    <p class="minutes-missing hidden">A number is required.</p>
+                    <p class="minutes-range hidden">Enter a number between 0 - 1,440.</p>
```

Comment:
suggest that we replace the hyphen with the word "and" to avoid hard-to-interpret line breaks (on my screen I got "Enter a number between 0" on one line and "-1,440" on another line)